File: apps/native-component-list/modules/expo-css-view/ios/ExpoCssViewModule.swift
import ExpoModulesCore

public class ExpoCssViewModule: Module {
  public func definition() -> ModuleDefinition {
    Name("ExpoCssView")

    View(ExpoCssView.self) {
      Prop("filter") { (view, filter: [[String: Any]]?) in
        view.setFilter(filter)
      }
      
      Prop("backdropFilter") { (view, backdropFilter: [[String: Any]]?) in
        view.setBackdropFilter(backdropFilter)
      }
    }
  }
}

File: apps/native-component-list/modules/expo-css-view/android/src/main/java/expo/modules/cssview/ExpoCssViewModule.kt
package expo.modules.cssview

import expo.modules.kotlin.modules.Module
import expo.modules.kotlin.modules.ModuleDefinition
import java.net.URL

class ExpoCssViewModule : Module() {
  override fun definition() = ModuleDefinition {
    Name("ExpoCssView")

    View(ExpoCssView::class) {}
  }
}
